pycypher/osx_build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e -x
+
+if [ "$TRAVIS" != "true" ] || [ "$CI" != "true" ]; then
```

Comment:
Perhaps also rename the script or move it to a "travis" or "ci" directory?

Reply:
Ok, I can rename the whole `utils/` folder to `ci/`

Reply:
Better might be to create a folder at the root, as these scripts could be used for building all packages through the CI (not just python).